index.html

```diff
@@ -265,6 +278,11 @@ <h2 class="bannerbox__title">I need a ride</h2>
                             </label>
                             <small>Yes, please stay in touch!</small>
                         </div>
+						<div class="form-group">
+							<script src='https://www.google.com/recaptcha/api.js'></script>
+							<div class="g-recaptcha" data-sitekey="YOUR_KEY_1"></div> <!--Generate your site owner key (http://www.google.com/recaptcha/intro/) and add here-->
```

Comment:
We're going to need to generate a site key for you. I'll get back to you about this.